deploy.sh

```diff
@@ -152,8 +161,12 @@ svn update
 
 svn status
 
-echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$DRY_RUN" == true ]]; then
+  echo "➤ Debug mode: Files not committed."
```

Comment:
Probably just an oversight but "Debug mode" is no longer mentioned anywhere
```suggestion
  echo "➤ Dry run: Files not committed."
```